### PR TITLE
Add ZDoom cask

### PR DESCRIPTION
**Pull request by vincecima**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/zdoom.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'zdoom' do
+  version '2.8.1'
+  sha256 '4143e6a484c670f26ce56ef758313c7ca18455368e825665023b14fa3b8ee934'
+
+  url 'http://zdoom.org/files/zdoom/2.8/zdoom-2.8.1.dmg'
```
Comment (vitorgalvao) [inline Casks/zdoom.rb]: `url "http://zdoom.org/files/zdoom/#{version.major_minor}/zdoom-#{version}.dmg"`

**Comment by vitorgalvao** [comment]
Welcome back, @vincecima! Long time no see. It’s been a few years since I remember you last contributing here. Good to see you. Hope all has been well.

File: Casks/zdoom.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/zdoom.rb]: See [`version` docs](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/version.md) for more information. We didn’t have fancy version methods back then.

**Comment by vincecima** [comment]
@vitorgalvao thanks, you are to fast!

**Pull request merged**